wasm/src/lib.rs

```diff
@@ -1,33 +1,133 @@
 mod wasm_builtins;
 
+extern crate js_sys;
+extern crate num_bigint;
 extern crate rustpython_vm;
 extern crate wasm_bindgen;
 extern crate web_sys;
 
-use rustpython_vm::VirtualMachine;
+use num_bigint::BigInt;
 use rustpython_vm::compile;
-use rustpython_vm::pyobject::AttributeProtocol;
+use rustpython_vm::pyobject::{self, IdProtocol, PyObjectRef, PyResult};
+use rustpython_vm::VirtualMachine;
 use wasm_bindgen::prelude::*;
 use web_sys::console;
 
+fn py_str_err(vm: &mut VirtualMachine, py_err: &PyObjectRef) -> String {
+    vm.to_pystr(&py_err)
+        .unwrap_or_else(|_| "Error, and error getting error message".into())
+}
+
+fn py_to_js(vm: &mut VirtualMachine, py_obj: &PyObjectRef) -> JsValue {
```

Comment:
Could we re-use the stdlib/json.rs module here?

Reply:
Oh, I hadn't even thought of that. I'll try that.

Reply:
Yep, it works! That's a lot simpler than before.